qemu/tests/cfg/virtio_mem_numa_basic.cfg

```diff
@@ -0,0 +1,43 @@
+- virtio_mem_numa_basic:
+    only Linux
+    only Host_RHEL.m9
+    only RHEL.9
+    no s390x
+    type = virtio_mem_numa_basic
+    virt_test_type = qemu
+    login_timeout = 240
+    threshold = 0.025
+    smp = 8
+    vcpu_maxcpus = 16
```

Comment:
Included, thanks!

Reply:
Please help put the aarch64 attribute below, otherwise, the setting will be overwritten by the vcpu_maxcpus below.
like this: 

```
    vcpu_maxcpus = 16
    aarch64:
        vcpu_maxcpus = ${smp}
```

everything else looks great to me, very clear and concise, thanks for your effort :)